utils/getDateByInterval.js

```diff
@@ -0,0 +1,15 @@
+/**
+ * Get the date at an interval from today's date.
```

Comment:
By interval do you mean the number of days after today's date? Or does this function return multiple days at an interval (i.e. every two days after today's date)

Reply:
it returns the date object of (currentDate + interval)